node-js-basics-101/node_modules/xmlhttprequest-ssl/tests/test-headers.js

```diff
@@ -0,0 +1,76 @@
+var sys = require("util")
+  , assert = require("assert")
```

Comment:
__[jshint]__
* Comma warnings can be turned off with 'laxcomma'.
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->